aestrivex/bctpy
# test/build update and random seeds

QoL updates:

- Use tox so that developers can easily run all tests on all versions
- Test against more versions of python
- Use pip instead of conda
  - when conda was first used here, it represented a large gain in build speed, but the opposite is now true
  - pip is the lowest common denominator and simplifies the build a lot
- `__version_info__` for proper version sorting, if double digits are ever hit
- wrote and skipped trivial tests for a couple of functions I found to have bugs but don't know how to fix (see #68 )

Random seeds (see #67 ): all functions which directly called np.random functions, and all
functions which call those functions (I think), now accept a `seed` kwarg.

- By default, the global numpy random number generator is used
- Otherwise, the passed RandomState instance or a new one with the given seed is used
- See utils.miscellaneous_utilities.get_rng for details
